src/Features/Core/Portable/ExternalAccess/UnitTesting/API/NewUnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -53,7 +53,9 @@ public void Reanalyze()
 
         var metadata = new UnitTestingIncrementalAnalyzerProviderMetadata(
             analyzerName,
+#if false // Not used in unit testing crawling
             highPriorityForActiveFile: false,
+#endif
```

Comment:
I am surprised by this. My assumption was that the crawler always prioritized active files. I wonder if we are leaving some responsiveness on the table by setting this to false.

Is there any harm in preserving this so that we could experiment with switching this to true down the line?

Reply:
i'm preservinb this by keeping the code in teh ifdef around.  when you guys take over this, you can totally try flipping the switch on this.  but i'm not a fan of speculatively keeping this around with all the complexity when it's not even being used currently, and all this code is going to be 'legacy' and ideally replaced with something better asap :)

Reply:
Fair enough. So long as the code is available, we can experiment with it.

The only reason I was asking is that this could be an easy perf win that users would notice (especially when adding tests within larger solutions). It would be faster to experiment and improve the user experience with the existing code than to say that we need to first move over to a pull-based model before we can do that :)

Reply:
i think that's totally fine :)  that's why i'm leaving the code in there.  you can def add/augment it going forward to try things out.  my hope though is that we can start with an extremely small and lightweight kernel that is easy to maintain versus having to pull in everything in the crawler space.